packages/remix/test/build.test.ts

```diff
@@ -0,0 +1,53 @@
+import assert from 'assert';
+import { join } from 'path';
+import { NodejsLambda } from '@vercel/build-utils';
+import { build, prepareCache } from '../src';
+
+const fixture = (name: string) => join(__dirname, 'fixtures', name);
+
+describe('build()', () => {
+  jest.setTimeout(10 * 60 * 1000);
```

Comment:
Because `jest.setTimeout` applies to the whole file, can we move it out of the `describe` block to make it clearer it applies beyond this `describe` block?